source/src/molecular-unfolding/cdk/stack-main.ts

```diff
@@ -76,6 +76,14 @@ export class MainStack extends SolutionStack {
       constraintDescription: 'Any printable ASCII character ranging from the space character (\u0020) through the end of the ASCII character range',
     });
 
+    const quickSightRoleArn = new CfnParameter(this, 'QuickSightRoleArn', {
+      type: 'String',
+      description: 'QuickSight IAM Role Arn, find it from https://us-east-1.quicksight.aws.amazon.com/sn/admin#aws',
```

Comment:
I think you should remove `find it from https://us-east-1.quicksight.aws.amazon.com/sn/admin#aws`

Reply:
removed